mm2src/mm2_bitcoin/spv_validation/src/helpers_validation.rs

```diff
@@ -312,7 +321,63 @@ pub struct BlockHeaderVerificationParams {
     pub difficulty_check: bool,
     pub constant_difficulty: bool,
     pub difficulty_algorithm: Option<DifficultyAlgorithm>,
-    pub genesis_block_header: String,
+    // Where to start fetching block height from. 0 will be the default value if starting_block_header_height is None
+    pub starting_block_header_height: Option<u64>,
+    // Use (starting_block_header_height +1) header hex if starting_block_header_height is set else use genesis block
+    // header hex.
+    pub starting_block_header_hex: String,
+    // retarget_block_header_height is needed if starting_block_header_height is set.
+    pub retarget_block_header_height: Option<u64>,
+    // retarget_block_header_hex is needed if starting_block_header_height is set. (2016)
+    pub retarget_block_header_hex: Option<String>,
+}
+
+#[derive(Debug, Default, Clone, Deserialize)]
+pub struct SPVConf {
+    // Determine if spv proof should be enable. Default to false.
+    pub enable_spv_proof: bool,
+    // Limit of block headers allowed to stored in db. No limit will be set if max_stored_block_headers is None.
+    max_stored_block_headers: Option<u64>,
+    // Where to start fetching block height from. 0 will be the default value if starting_block_header_height is None
+    starting_block_header_height: Option<u64>,
+    /// The parameters that specify how the coin block headers should be verified. If None and enable_spv_proof is true,
+    /// headers will be saved in DB without verification, can be none if the coin's RPC server is trusted.
+    pub verification_params: Option<BlockHeaderVerificationParams>,
+}
```

Comment:
This is related to mm2 coin config, it shouldn't be part of `mm2_bitcoin` crates.